go/interpreter/lfvm/lfvm.go

```diff
@@ -16,10 +16,26 @@ import (
 	"github.com/Fantom-foundation/Tosca/go/tosca"
 )
 
+// Config provides a set of user-definable options for the LFVM interpreter.
+type Config struct {
+	// TODO: add options for code cache size and other configuration options
+}
+
+// NewInterpreter creates a new LFVM interpreter instance with the official
+// configuration for production purposes.
+func NewInterpreter(Config) (*lfvm, error) {
```

Comment:
since this is not an interface and it will be called in a context where other interpreters will also be instantiated, what do you think of `NewLfvmInstance`  instead ?

Reply:
It is in the package `lfvm` so it is most likely called as `lfvm.NewInterpreter(lfvm.Config{...})`. So the term `lfvm` already shows up twice. An example can be found [here](https://github.com/Fantom-foundation/Sonic/blob/8e5ede0624be3df10324f235e9058b6eef63d2ee/opera/rules.go#L32).